test/jobs/test_align/test_bam_cram.py

```diff
@@ -215,20 +206,15 @@ def test_indexes_bam_if_index_does_not_exist(self, tmp_path: Path):
         assert len(align_jobs) == 1
 
         cmd = get_command_str(align_jobs[0])
-        file = re.escape(f'SAMPLE1.bam')
+        file = re.escape('SAMPLE1.bam')
 
         # Test sorts and indexes alignment input,
         assert re.search(
-            (
-                fr'samtools sort \${{BATCH_TMPDIR}}/inputs/\w+/{file}'
-                r' .* > \$BATCH_TMPDIR/sorted\.bam'
-            ),
+            (fr'samtools sort \${{BATCH_TMPDIR}}/inputs/\w+/{file}' r' .* > \$BATCH_TMPDIR/sorted\.bam'),
```

Comment:
Here's another category of these (probably rare):

```suggestion
            (fr'samtools sort \${{BATCH_TMPDIR}}/inputs/\w+/{file} .* > \$BATCH_TMPDIR/sorted\.bam'),
```